src/lib/core/hooks/variableCoverage.ts

```diff
@@ -0,0 +1,112 @@
+import { useMemo } from 'react';
+
+import { CompleteCasesTable } from '../api/data-api';
+import {
+  VariableCoverageTableRow,
+  VariableSpec,
+} from '../components/VariableCoverageTable';
+import { Filter } from '../types/filter';
+
+import { useEntityCounts } from './entityCounts';
+
+/**
+ * Returns an array of data to be rendered in a VariableCoverageTable
+ * @param completeCases
+ * @param filteredEntityCounts
+ * @param variables
+ * @param outputEntityId
+ * @returns A VariableCoverageTableRow[]
+ */
+export function useVariableCoverageTableRows(
```

Comment:
Something to discuss at tomorrow's meeting and probably not here...

What's the difference between a hook and a (memoized) utility function?

Reply:
Sure - let's add that to the agenda!